### PR TITLE
fix v3.0.0 ci

### PR DESCRIPTION
**Pull request by zhaohaifei**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by Sophie-Xie — APPROVED** [review]

File: tests/Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ install-deps:
 install-nebula-py: install-deps
 	git clone --branch master https://github.com/vesoft-inc/nebula-python $(CURR_DIR)/nebula-python
 	cd $(CURR_DIR)/nebula-python \
-		&& pip3 install --user . -i $(PYPI_MIRROR) --upgrade
+		&& pip3 install --user --force-reinstall . -i $(PYPI_MIRROR)
```
Comment (darionyaphet) [inline tests/Makefile:37]: so we need reinstall nebula python each time?

Comment (zhaohaifei) [reply tests/Makefile:37]: I think so. Some runners will keep the previous nebula-python, resulting in a version mismatch.

Comment (HarrisChu) [reply tests/Makefile:37]: github runner would cache $HOME, and we install python package with `--user` option...

Comment (zhaohaifei) [reply tests/Makefile:37]: yeah.

**Pull request closed without merging**